cmd/channel/distributed/dispatcher/main.go

```diff
@@ -77,8 +77,21 @@ func main() {
 		logger.Fatal("Failed To Load Sarama Settings", zap.Error(err))
 	}
 
+	client.InitSaramaConfig(saramaConfig)
+	saramaConfig.ClientID = constants.Component
+
 	// Update The Sarama Config - Username/Password Overrides (EnvVars From Secret Take Precedence Over ConfigMap)
-	client.UpdateSaramaConfig(saramaConfig, constants.Component, environment.KafkaUsername, environment.KafkaPassword)
```

Comment:
This function was too specific. Got rid of it and changed its usages with using more generic `UpdateSaramaConfigWithKafkaAuthConfig`